web/src/js/components/FlowView/Messages.jsx

```diff
@@ -68,17 +68,18 @@ function ResponseLine({ flow, readonly, updateFlow }) {
 
 const Message = connect(
     state => ({
-        flow: state.ui.flow.modifiedFlow || state.flows.byId[state.flows.selected[0]],
+        flow: state.flows.byId[state.flows.selected[0]],
```

Comment:
in case that we just edit either the response or the request, this will work. if we want (and i think we want that) we can edit repsonse and request simultaneously than this is not possible